tests/providers/google/cloud/operators/test_dataproc.py

```diff
@@ -434,7 +591,10 @@ def test_execute(self, mock_hook):
             request_id=REQUEST_ID,
             impersonation_chain=IMPERSONATION_CHAIN,
         )
-        op.execute(context={})
+        op.execute(context=self.mock_context)
+
+        # Test whether xcom push occurs before polling for job
+        self.extra_links_manager_mock.assert_has_calls(self.extra_links_expected_calls, any_order=False)
```

Comment:
@turbaszek Have added checks for xcom_push before wait_for_job too. Please take a look.